### DISCUSSION TITLE
Number of processes

### DISCUSSION THREAD

**Discussion by iHeadRu** [roehling/postsrsd#151] [Q&A] [answered]
Hi!

I tried postsrd on FreeBSD from package repository, Current version there is 1.10 :(

I set up server like in [https://github.com/roehling/postsrsd/discussions/76#discussion-50695](https://github.com/roehling/postsrsd/discussions/76#discussion-50695).
It works.
I worry about number of processes. 
ps -aux | grep postsrsd | wc -l 
gives me 20.

I tried to read code and not found limit on forks count.

**Comment by roehling** [comment, marked as answer]
The number of processes is determined by the number of cleanup daemons executed by Postfix. Also, PostSRSd will linger for a while and wait for additional requests before it terminates the children. The default timeout in version 1.x is 30 minutes.
